extensions/parquet/table/src/test/java/io/deephaven/parquet/table/S3ParquetTestBase.java

```diff
@@ -520,52 +520,12 @@ public void indexByLongKey() {
     @Test
     public void testReadWriteUsingProfile() throws IOException {
         final Table table = TableTools.emptyTable(5).update("someIntColumn = (int) i");
-        Path tempConfigFile = null;
-        Path tempCredentialsFile = null;
+        // Create temporary config and credentials file and write correct credentials and region to them
+        final Path tempConfigFile = Files.createTempFile("config", ".tmp");
+        final Path tempCredentialsFile = Files.createTempFile("credentials", ".tmp");
         try {
```

Comment:
This is the only test which verifies that we are reading data from a config file and failing if bad data.
I would prefer to not remove the test and delay the localstack upgrade.

Reply:
I've added back the test in a new form; I updated the comment. Ultimately, we should try to be more specific with our tests and not test multiple things in a single test. As far as the earlier images were concerned, localstack was failing based on the region and MinIO was failing based on the keys.